src/App.css

```diff
@@ -1,23 +1,26 @@
 body {
-  background: url("../public/wall.jpg");
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 100%;
+
+background: url(../public/wallpaper.jpg)  no-repeat  center center;
+background-size: cover;
+display: flex;
+flex-direction: row;
```

Comment:
This is not needed. It row is the default for flex-direction

Reply:
removed